src/Repository/AdherentRepository.php

```diff
@@ -525,4 +526,27 @@ public function countByGenderManagedBy(Adherent $referent): array
             ->getArrayResult()
         ;
     }
+
+    public function countCommitteeSupervisorsByGenderForReferentManagedArea(Adherent $referent): array
```

Comment:
I propose rename this method `countSupervisorsByGenderForReferentTags(array $referentTags)`:
 - without `Committee` because already in Committee Repository
 - without `ReferentManagedArea` because you use the ReferentTags in your query
what do you think?

Reply:
Ok to remove `Committee` et I don't agree about `ReferentManagedArea` because it's supervisors count not of all committees, but committees in referent managed area. 
I can rename to `countSupervisorsByGenderForReferent`.